symplyphysics/laws/thermodynamics/isochoric_and_isobaric_heat_capacities_of_homogeneous_substance.py

```diff
@@ -0,0 +1,67 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols,
+)
+
+# Description
+## Mayer's relation is the relation between heat capacity at constant pressure and heat
+## capacity at constant volume.
+
+# Law: C_p - C_V = V * T * (alpha_V)**2 / beta_T
+## C_p - heat capacity at constant pressure
+## C_V - heat capacity at constant volume
+## V - volume
+## T - temperature
+## alpha_V - [thermal expansion coefficient](../../definitions/volumetric_coefficient_of_thermal_expansion.py)
+## beta_T - isothermal compressibility
```

Comment:
What's this?

Reply:
I think it is easier to create the definition file and give a link to it here

Reply:
Ok. Put TODO comment, in this case.

Reply:
Opened #586

Reply:
Merged. You can merge with it and put a link to this parameter definition.

Reply:
Yes, that was my intention